GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -44,6 +44,18 @@ class ChatViewModel: EngagementViewModel, ViewModel {
 
     private var pendingMessages: [OutgoingMessage] = []
     private var isViewLoaded: Bool = false
+    private (set) var isChoiceCardInputModeEnabled: Bool = false
+    private (set) var siteConfiguration: CoreSdkClient.Site?
+
+    var mediaPickerButtonVisibility: MediaPickerButtonVisibility {
+        guard let site = siteConfiguration else { return .disabled }
+        guard site.allowedFileSenders.visitor else { return .disabled }
+        guard environment.getCurrentEngagement() != nil else {
+            return .enabled(.enagagementConnection(isConnected: false))
+        }
+        guard !isChoiceCardInputModeEnabled else { return .enabled(.choiceCard) }
+        return .enabled(.enagagementConnection(isConnected: environment.getCurrentEngagement() != nil))
```

Comment:
Will it work correctly on `connecting`/`transferring` states?

Reply:
It did not rely on `connecting/transferring` previously, so why it should now? Let's check it during exploratory testing.